src/Application.cpp

```diff
@@ -392,6 +392,70 @@ namespace weatherserver {
     location.setIsAddingWeatherToAddressSpace(false);
   }
 
+  /*
+  Checking if locations number in the server model matches with passed value. Trying to adjust if its different.
+  Returns true if:
+      - model locations number was successfuly read and already matches with passed value;
+      - value is different, but adjustment (write) was successful.
+  Returns false if:
+      - couldn't read existing value from the model;
+      - existing value was different and adjustment (write) failed.
+  */
+  static bool validateLocationsNumberInTheModel(UA_Server& server, const CountryData& country, uint32_t valueToValidate) {
+
+    bool validationFlag = false;
+
+    //crafting NodeID for our locations number attribute inside the information model
+    std::string locationsNumberAttributeString = std::string(CountryData::COUNTRIES_FOLDER_NODE_ID) + "."
+      + country.getCode() + "." + std::string(CountryData::BROWSE_LOCATIONS_NUMBER);
```

Comment:
It seems that construction of this string should be part of `CountryData` class.

Reply:
Makes sense so we don't recreate it twice. Added to TODO. Maybe add more nodeIDs to approprate classes in case need to reuse.